chainlink-testing-framework/setup-run-tests-environment/action.yml

```diff
@@ -48,7 +48,7 @@ runs:
 
     # Setup AWS cred and K8s context
```

Comment:
If you update this file you will want to bump the run-tests action to use the expected new tag version in this PR for it to be reflected there. Either that or you need to merge this PR and then open another one to do the bump after the new tag is created. Its a fun chicken and the egg problem caused by actions using actions and them being in the same repo.

Reply:
Ah I didn't see that.

Alternative: I think it's possible to point it the the action directly rather than using a tag (`uses: ./chainlink-testing-framework/setup-run-tests-environment`). So it's always using the action at the same tag as the `setup-run-tests-environment`.

Reply:
Spoke with @HenryNguyen5. Although the local reference is possible, then the action would have to checkout it's repository itself which is slow. So actions like these are better off:
1. Being nodejs actions, not composite
2. Bundled together at build time so they're always in sync

If I take this on, these would be migrated to the `.github` repo as well. Thoughts?

Reply:
Whatever works. If you want to get these all bumped and merged then we can bump the reference in run-tests for now we can do that. I am assuming it will take some time to build out the node version for these and get it tested across all our use cases.

Reply:
I will be making changes for these as well in the next couple of days to fix the usage for the docker tests so the use cases will be changing a bit as well.

Reply:
> I will be making changes for these as well in the next couple of days to fix the usage for the docker tests so the use cases will be changing a bit as well.

For now, does it make sense to:
1. Merge these changes
2. Release these dependency bumps when you make a release with your planned updates

And we'll migrate these over to nodejs+.github as part of [RE-2003](https://smartcontract-it.atlassian.net/browse/RE-2003).

Reply:
Sounds good